messages/pt-PT.json

```diff
@@ -0,0 +1,15 @@
+{
+  "profile-form.save-changes": "Guardar alterações",
+  "profile-form.field.firstName": "Nome",
+  "profile-form.field.lastName": "Apelido",
+  "profile-form.field.gender": "Género",
+  "profile-form.field.custom-gender": "Género Personalizado",
+  "profile-form.field.birthDate": "Data de nascimento",
+  "profile-form.field.homePhone": "Telefone de casa",
+  "profile-form.field.businessPhone": "Telefone do Trabalho",
+  "profile-form.field.corporateDocument": "Documento Comprovativo de Trabalho",
+  "profile-form.field.corporateName": "Nome da Entidade Empregadora",
+  "profile-form.field.stateRegistration": "Registro Civil",
+  "profile-form.field.tradeName": "Nome fantasia",
```

Comment:
You should set a standard with _Sentence case_ or _Pascal Case_ but not both.

Reply:
instead of _Pascal Case_, shouldn't it be _Camel Case_?

Reply:
This is actually a copy of the other files' keys. I don't know if it's worth to change it now, since I'll need to change in all the other files and in its usage throughout the code. I'll create an issue to track this problem and solve it in a refactor later. What do you guys think?

Reply:
@arthurepc Yes, it's interesting to me.

Reply:
Fine with me, creating a dedicated issue.